### PR TITLE
Fix Maexxna's portal rotation.

### LINKED ISSUE #217: Naxxramas - Maexxna's Naxxramas Portal
Screenshot: https://gyazo.com/d9fc5a258444a8a8bede96780af60d67

**Description**:

After killing Maexxna, a 'portal' appears which funnily enough doesn't look like a portal, so I guess we could just call it a transporter. 

**Current behaviour**: Tell us what happens

After killing Maexxna, it spawns, but in the wrong position.

**Expected behaviour**: Tell us what should happen instead

After killing Maexxna, it should spawn near the wall. Here's a video where you can see the correct positioning: https://youtu.be/-709sus1gIg?t=6m59s

**Steps to reproduce the problem**:

1. Kill all of Arachnid Quarter
2. Observe

**Branch(es)**: 0.x / 1.x / master (Specify the branch(es) affected by this issue) 

**AC hash/commit**:  commit 7e4328b588c44d916a530a629d282b5c6a5ab0c8

**Operating system**:  Windows

**Modules**:  Are you using modules? if yes, list them

**Other Customizations**: Are you use core/db patches or other customizations? If yes please specify them here.

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing c++ create a Pull Request)

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/38823915-naxxramas-maexxna-s-naxxramas-portal?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F40032087&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by FALL1N1**
**Changes proposed:**
 Hackfix for the sunwell's gameobject rotation hackfix.

**Target branch(es):**
all?

**Issues addressed:**
Closes #217

**Tests performed:**
none

**How to test the changes:**
build, kill maexxna and check portal location

**Known issues and TODO list:**
check if guids are the same with azeroth's wdb, mine could be different (asd.sql)
if it doesn't work let me know i will look for previous commits from my core since this might not be the only one

### CODE REVIEW

**Comment by FALL1N1** [comment]
p.s maybe we can use gameobject_addon for the "if (addon && addon->invisibilityType == INVISIBILITY_GENERAL && addon->InvisibilityValue == 0)" check but meh, up to whoever decides to merge

**Comment by Nefertumm** [comment]
Hi, thanks for your contribution, but please read this to avoid travis problems. https://github.com/azerothcore/azerothcore-wotlk/wiki/Dealing-with-SQL-files

File: data/sql/updates/pending_db_world/asd.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+-- maexxna portals
+REPLACE INTO `gameobject` (`guid`, `id`, `map`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `VerifiedBuild`, `zoneId`, `areaId`) VALUES
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/asd.sql]: ZoneId and areaId doesn't exist in our creature table. Fix this and read the wiki that i commented above. Thanks!!

**Comment by BarbzYHOOL** [comment]
Yes, the "Dealing with SQL files" page looks long and annoying to read but in fact it takes 2 min to make the SQL the correct way. (We probably need to make it shorter or divided into several pages)

File: data/sql/updates/pending_db_world/asd.sql (hunk 1 above)
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/asd.sql]: ZoneId and areaId doesn't exist in our creature table. Fix this and read the wiki that i commented above. Thanks!!

**Comment by FALL1N1** [comment]
i have dumped it as-is from my db so someone can test it, i'll rewrite it later

**Comment by Nefertumm** [comment]
That's fine, thanks. We will wait your rewrite.

**Comment by FALL1N1** [comment]
done, still i'm not sure about the guids on azeroth but if naxx guids are same as sunwell's it should work fine

**Comment by Nefertumm** [comment]
GUIDs are okay.

**Comment by Dreaxxx** [comment]
Work fine

**Comment by talamortis** [comment]
Thanks for testing people will merge 👍  thanks for your contribution aswell @FALL1N1

**Pull request merged**